docker-compose.yml

```diff
@@ -39,7 +39,7 @@ services:
       db:
         condition: service_started
       app:
-        condition: service_healthy
+        condition: service_started
```

Comment:
I found my `worker` service always had to be started manually locally, so I made this change to fix that.

Reply:
I think this was originally implemented because if app wasn't running worker occasionally crashed on start becauese the migrations weren't applied. idk though. If it works for you maybe that's not true

Reply:
It could have been a false positive for me if I had already applied migrations locally, then. I just noticed that my worker _never_ started on it's own, I always had to go start it manually, so I figured this was the cause. ¯\\\_(ツ)_/¯